src/Navbar.js

```diff
@@ -93,13 +124,19 @@ class Navbar extends React.Component {
   }
 
   getChildContext() {
-    const { bsClass, expanded } = this.props;
+    const { bsClass, expanded, onSelect, toggleOnSelect } = this.props;
+    const mobileToggle = () => {
+      if (typeof(window) !== 'undefined' && window.innerWidth < 768) {
```

Comment:
I don't think we need this check.  it's too brittle anyway as a user can change the navbar collapse breakpoint to whatever they want so hard coding a pixel width here isn't ideal. 

I do think that it illustrates  a more general problem with this feature tho,  you don't really want to be toggling expanded state while the expansion is not relevant, however there isn't really any way to know that from JS